### PR TITLE
Add CI workflows

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR adds CI workflows to
- validate PRs that are being merged to main
- build an Android APK
- publish app to TestFlight

TODO:
- [x] Add secrets to repo
- [ ] Create a development provisioning profile for `l-breez`
- [ ] Add `l-breez` to TestFlight (Requires admin privileges - @roeierez)

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          path: 'lbreez'
+
+      # TODO: Liquid - Revert once breez-sdk dependency is removed
+      - name: 🏗️ Setup breez-sdk repository
+        uses: actions/checkout@v4
+        with:
+          repository: 'breez/breez-sdk'
+          ssh-key: ${{secrets.REPO_SSH_KEY}}
+          path: 'breez-sdk'
+          ref: 'fca80043d011f213741ddae93973ed264a8b6ddb'
```
Comment (dangeross) [inline .github/workflows/CI.yml:33]: Would it be worth adding this as a `breez_sdk_ref` input? Maybe a specific branch would be better

File: .github/workflows/build-android.yml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+      - name: 🏗️ Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: breez-liquid-sdk/lib/
+          cache-all-crates: true
+
+      - name: 📦 Install Breez Liquid SDK dependencies
+        run: |
+          cd breez-liquid-sdk/lib/bindings/langs/flutter/
```
Comment (dangeross) [inline .github/workflows/build-android.yml:101]: could be a `working-directory`

File: .github/workflows/build-ios.yml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+      - name: 🏗️ Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: breez-liquid-sdk/lib/
+          cache-all-crates: true
+
+      - name: 📦 Install Breez Liquid SDK dependencies
+        run: |
+          cd breez-liquid-sdk/lib/bindings/langs/flutter/
```
Comment (dangeross) [inline .github/workflows/build-ios.yml:107]: could be a `working-directory`

File: .github/workflows/build-ios.yml [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        if: github.event_name != 'release'
+        uses: actions/upload-artifact@v3
+        with:
+          name: release-iOS
+          path: lbreez/build/ios/iphoneos
+
+      - name: 📱 Publish to TestFlight
+        run: |
+          altool="$(dirname "$(xcode-select -p)")/Developer/usr/bin/altool"
+          ipa="$PWD/lbreez/build-output/ios/c_breez.ipa"
```
Comment (dangeross) [inline .github/workflows/build-ios.yml:230]: Not sure what it should be
```suggestion
          ipa="$PWD/lbreez/build-output/ios/l_breez.ipa"
```

File: .github/workflows/CI.yml (hunk 1 above)
Comment (erdemyerebasmaz) [reply .github/workflows/CI.yml]: Changed it target `flutter_rust_bridge_v2` by default but this change will only be reflected once workflow file is merged to `main`.

**Pull request merged**